pramen-py/tests/test_metastore/test_metastore.py

```diff
@@ -368,33 +367,39 @@ def test_metastore_get_latest(
     Our data has partitions between 2022-03-23 and 2022-04-26.
     """
 
-    metastore = MetastoreReader(
-        spark=spark,
-        config=load_and_patch_config.metastore_tables,
-        info_date=info_date,
-    )
-    if exc:
-        with pytest.raises(exc, match=exc_msg):
-            metastore.get_latest(
-                "table1_sync",
-                until=until,
-            )
-    else:
-        actual = metastore.get_latest(
-            "table1_sync",
-            until=until,
-        )
-        expected = spark.read.parquet(
-            load_and_patch_config.metastore_tables[0].path
+    for format_ in TableFormat:
```

Comment:
It is better to use @pytest.mark.parametrize and set two formats. So you'll have better handling of your tests by pytest